search/typesenseGoandJS/README.md

```diff
@@ -0,0 +1,21 @@
+INSTALL TYPESENSE VIA DOCKER:
+docker pull typesense/typesense:0.22.2
+
+MAKE DIRECTORY:
+mkdir /tmp/typesense-data
+
+START UP THE TYPESENSE SERVER:
+docker run -p 8108:8108 -v/tmp/typesense-data:/data typesense/typesense:0.22.2 --data-dir /data --api-key=xyz --enable-cors
+
+install javascript client library:
+npm install --save typesense
+npm install --save @babel/runtime
+
+install go typesense client library:
+go get github.com/typesense/typesense-go
+
+index files:
+go run index.go
```

Comment:
still get this error:
panic: status: 409 response: {"message": "A collection with name `featuredata` already exists."}

goroutine 1 [running]:
main.main()
        C:/Users/sam/desktop/dev/featureform/embeddings/search/typesenseGoandJS/index.go:54 +0x8a5
exit status 2

perhaps have it check if it exists first and if it does overwrite or delete